libs/aws/langchain_aws/llms/bedrock.py

```diff
@@ -969,6 +986,12 @@ def validate_environment(cls, values: Dict) -> Dict:
                 "Please use `from langchain_community.chat_models import BedrockChat` "
                 "instead."
             )
+        if model_id.startswith("cohere.command-r"):
```

Comment:
May I check why it's not supported here?

Reply:
command-r has been excluded from BedrockLLM for the same reason as claude-3. Specifically, command-r differs from the standard command model in terms of output format and behavior. Due to these differences, it requires specialized handling, which is why it has been excluded from the general BedrockLLM class.